backend/app/categories/controller.py

```diff
@@ -87,22 +97,28 @@ async def update_item_route(category_id: CategoryEnum, item_id: ObjectID,
 
     # checking if request body is of correct type
     if not isinstance(item, classof[category_id]):
-        raise HTTPException(
-            status_code=400, detail=f'Item does not match {category_id} schema')
+        raise HTTPException(status_code=400,
+                            detail=f'Item does not match {category_id} schema')
     _res = await db[category_id]["data"].update_one({"_id": item_id}, {"$set": item.dict()})
     if _res.matched_count == 0:
         raise HTTPException(
-            status_code=400, detail=f'ObjectID {item_id} not found in {category_id}')
+            status_code=400,
+            detail=f'ObjectID {item_id} not found in {category_id}')
+
+    # add id to the item
+    item.id = item_id
     return ItemInResponse(data=item.dict())
 
 
-@router.post("/{category_id}/rate/{item_id}", response_model=ResponseBase, tags=["user", "rate"])
+@router.post("/{category_id}/rate/{item_id}",
+             response_model=ResponseBase, tags=["user", "rate"])
 async def rate_item_route(category_id: CategoryEnum, item_id: ObjectID, _rating: Dict[str, float],
                           token: dict = Depends(verify_token),
                           db: AsyncIOMotorClient = Depends(get_database)) -> ResponseBase:
     """"Rate a given item"""
     rating = _rating.get("rating", None)
     item = await db[category_id]["data"].find_one({"_id": item_id})
+    print(repr(category_id))
```

Comment:
I think you forgot to remove this debugging line

Reply:
Yeah! Sorry

Reply:
How can i remove now ?